filter_plugins/ec2_custom.py

```diff
@@ -1,5 +1,15 @@
 from ansible import errors
 
+# list all known app component names.
+VALID_APP_COMPONENT_NAMES = ["frontend",
```

Comment:
at runtime only the options for the specific project are relevant.  The args can be supplied to the filter via group_vars